src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const tHeadTags = document.querySelectorAll('th');
```

Comment:
Consider checking if `tHeadTags` is not empty before adding event listeners to avoid potential errors if the table headers are not present.